resources/Dockerfile.phan

```diff
@@ -0,0 +1,11 @@
+FROM php:7.2-buster
```

Comment:
We should probably run these on PHP 7.1 if possible, since that's the oldest we support.

Reply:
Sounds like a good plan.  I'll swap this out.